srcpkgs/nemo-compare/template

```diff
@@ -11,5 +11,6 @@ short_desc="Nemo compare extention"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/linuxmint/nemo-extensions"
-distfiles="https://github.com/linuxmint/nemo-extensions/archive/refs/tags/${version}.tar.gz"
-checksum=6a0b83e6a9129c7217c29fbc992109d8f6eedbe8f076da863c507c0005182404
+# distfiles="https://github.com/linuxmint/nemo-extensions/archive/refs/tags/${version}.tar.gz"
```

Comment:
is this needed?

Reply:
For 5.8.1, yeah, see https://github.com/linuxmint/nemo-extensions/tags.

(If you mean why use 5.8.1, it has this one fix: https://github.com/linuxmint/nemo-extensions/commit/d25b09fcd00c4970c70485fb50d3595d8e6c8ef8)